config/karma.base.js

```diff
@@ -59,7 +59,8 @@ const config = {
   // https://npmjs.org/browse/keyword/karma-preprocessor
   preprocessors: {
     'test/**/*.ts': ['webpack', 'sourcemap'],
-    'src/**/*.test.ts': ['webpack', 'sourcemap']
+    'src/**/*.test.ts': ['webpack', 'sourcemap'],
+    'integration/**/*.ts': ['webpack', 'sourcemap']
```

Comment:
Putting this in the base is going to affect every other product with an integration/ subdirectory, maybe just extend this in packages/vertexai/integration/karma.conf.js ?

Reply:
Fixed